index.html

```diff
@@ -137,12 +136,39 @@
     </div>
   </script>
 
+  <!-- Project List template -->
+  <script type="text/template" id="project-list-template">
+    <%
+      var classNames = [
+        "project-list",
+        thumbnailView ? "thumbnail-view" : "",
+        starred ? "starred" : "",
+        featured ? "featured" : "",
+      ].join(" ");
+    %>
+    <div id="<%- id %>" 
+          class="<%- classNames %>" 
+          data-title="<%- title %>"
+          data-issue-online-privacy-and-security="<%- belongsToOnlinePrivacyAndSecurityIssue %>"
+          data-issue-open-innovation="<%- belongsToOpenInnovationIssue %>"
+          data-issue-decentralization="<%- belongsToDecentralizationIssue %>"
+          data-issue-web-literacy="<%- belongsToWebLiteracyIssue %>"
+          data-issue-digital-inclusion="<%- belongsToDigitalInclusionIssue %>" >
+      <div class="content">
+        <a href="<%- link %>" target="_blank"><h2><%- title %></h2></a>
+        <h3><% if (creators) { %> <%- creators %> <% } %><% if (creators && timestamp)  %><% if (timestamp) { %> on <%- timestamp %> <% } %></h3>
+      </div>
+      <div class="hidden-meta"><%- tags %> <%- issues %> <%- program %> <%- type %></div>
+    </div>
+  </script>
```

Comment:
This template can be a lot simpler since most of the meta included in the code above are not needed for the favs page. (e.g., `data-*` attributes on the wrapper `<div>`, the `hidden-meta` `<div>`...etc)